File: Android applications/p0261intentfilter/java/com/korobeynikov/p0261intentfilter/ActivityDateEx.java
package com.korobeynikov.p0261intentfilter;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

import java.text.SimpleDateFormat;
import java.util.Date;

public class ActivityDateEx extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.date);
        SimpleDateFormat sdf=new SimpleDateFormat("EEE, MMM d, yyyy");
        String date=sdf.format(new Date(System.currentTimeMillis()));
        TextView tvDate=findViewById(R.id.tvDate);
        tvDate.setText(date);
    }
}

File: KotlinLessons/AndroidLessons/p0261intentfilter/java/ru/korobeynikov/p0261intentfilter/ActivityDateEx.kt
package ru.korobeynikov.p0261intentfilter

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.databinding.DataBindingUtil
import ru.korobeynikov.p0261intentfilter.databinding.ActivityDateBinding
import java.text.SimpleDateFormat
import java.util.*

class ActivityDateEx : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        val binding = DataBindingUtil.setContentView<ActivityDateBinding>(this, R.layout.activity_date)
        val sdf = SimpleDateFormat("EEE, MMM d, yyyy")
        val date = sdf.format(Date(System.currentTimeMillis()))
        binding.tvDate.text = date
    }
}